app/src/main/kotlin/nl/q42/template/logging/CrashlyticsAntilog.kt

```diff
@@ -0,0 +1,40 @@
+package nl.q42.template.logging
+
+import com.google.firebase.crashlytics.ktx.crashlytics
+import com.google.firebase.ktx.Firebase
+import io.github.aakira.napier.Antilog
+import io.github.aakira.napier.DebugAntilog
+import io.github.aakira.napier.LogLevel
+
+/** A value suitable for Crashlytics */
+private const val MAX_CHARS_IN_LOG = 1200
+
+class CrashlyticsAntilog : Antilog() {
+
+    private val logcatAntilog = DebugAntilog()
+
+    override fun performLog(
+        priority: LogLevel,
+        tag: String?,
+        throwable: Throwable?,
+        message: String?
+    ) {
+        // also send to logcat
+        logcatAntilog.log(priority, tag, throwable, message)
```

Comment:
this class is only used on release builds. Here we log debug as well, or am I missing something? I would prefer not logging debug or info to console in release builds.

Reply:
Agree, added a check